cdc/redo/manager.go

```diff
@@ -51,9 +51,10 @@ const (
 )
 
 const (
-	// supposing to replicate 10k tables, each table has one cached changce averagely.
+	// supposing to replicate 10k tables, each table has one cached change averagely.
 	logBufferChanSize = 10000
-	logBufferTimeout  = time.Minute * 10
+	// supposing to replicate 10k tables, each table has one resolvedTs change averagely.
+	flushBufferChanSize = 10000
```

Comment:
Can you estimate the memory consumption for each `ManagerImpl`?

Reply:
ok